integration_tests/v2/run_integration_tests.sh

```diff
@@ -0,0 +1,25 @@
+	while getopts v: flag
```

Comment:
minor thing. Could you line up your while loop and give it a bash shebang if it's bash or sh one if it's sh (bash is prolly fine): `#!/bin/bash`

Reply:
dunnit